media-sound/freac/freac-1.1~beta2.recipe

```diff
@@ -54,7 +54,7 @@ REQUIRES="
 	lib:libfaac$secondaryArchSuffix
 	lib:libfaad$secondaryArchSuffix
 	lib:libFLAC$secondaryArchSuffix
-	lib:libMAC$secondaryArchSuffix >= 5
+	lib:libMAC$secondaryArchSuffix >= 6
```

Comment:
I'm concerned that no recipe in the repo seems to use devel:libMAC, and yet the freac recipe is depending on lib:libMAC.
Could you clarify this?

Reply:
The Monkey's Audio library is loaded dynamically at runtime (using dlopen) which is why it is not required during build. fre:ac will run without it, but would be missing the Monkey's Audio codec then.

This would probably be a RECOMMENDS dependency if Haiku recipes supported that.